src/components/DataRepository/ActivityIndexSpecs.tsx

```diff
@@ -39,6 +39,38 @@ export const ActivityIndexSpecs = () => {
                   </td>
                 </tr>
               ))}
+              {pathway.map(({ property, type, description }) => (
+                <tr
+                  key={property}
+                  className="text-slate-900 dark:text-slate-100"
+                >
+                  <td className="whitespace-nowrap font-semibold py-4 pl-4 pr-3 sm:pl-6 md:pl-0">
+                    pathway.{property}
+                  </td>
+                  <td className="whitespace-nowrap py-4 px-3 text-base">
+                    {type}
+                  </td>
+                  <td className="whitespace-nowrap py-4 px-3 text-base">
+                    {description}
+                  </td>
+                </tr>
+              ))}
+              {patient.map(({ property, type, description }) => (
+                <tr
+                  key={property}
+                  className="text-slate-900 dark:text-slate-100"
+                >
+                  <td className="whitespace-nowrap font-semibold py-4 pl-4 pr-3 sm:pl-6 md:pl-0">
+                    pathway.{property}
```

Comment:
Incorrect, needs to be `patient.{property}`.

```suggestion
                    patient.{property}
```